selectel/resource_selectel_dbaas_datastore_v1_test.go

```diff
@@ -115,5 +231,245 @@ resource "selectel_dbaas_datastore_v1" "datastore_tf_acc_test_1" {
     ram = 4096
     disk = 32
   }
+  config = {
+	xmloption = "content"
```

Comment:
Please make indentation with two spaces instead of tab characters here and below